src/AutosizeInput.js

```diff
@@ -96,10 +96,12 @@ const AutosizeInput = React.createClass({
 		const inputStyle = Object.assign({}, this.props.inputStyle);
 		inputStyle.width = this.state.inputWidth + 'px';
 		inputStyle.boxSizing = 'content-box';
+		const { defaultValue, onChange, value } = this.props;
+		const inputProps = { defaultValue, onChange, value, placeholder: this.props.placeholder };
```

Comment:
My first thought was in fact to create a `placeholder` variable just like the others.  But there is the pre-existing `placeholder` variable that it would conflict with, so I did as I did in the interest of a minimal change.  Renaming the other `placeholder` var might be the best for consistency and future readability.